Wire-iOS/Sources/UserInterface/Components/MessagePreviewView.swift

```diff
@@ -148,7 +146,7 @@ final class MessageThumbnailPreviewView: UIView {
             }
         }
         else if message.isVideo, let fileMessageData = message.fileMessageData {
-            let imageIcon = NSTextAttachment.textAttachment(for: .videoMessage, with: .textForeground, and: .medium)!
+            let imageIcon = NSTextAttachment.textAttachment(for: .videoMessage, with: .textForeground)!
```

Comment:
The image here should be `.videoCall` according to the spec.

```suggestion
            let imageIcon = NSTextAttachment.textAttachment(for: .videoCall, with: .textForeground)!
```